spec/models/aws_engine_spec.rb

```diff
@@ -2,6 +2,81 @@
 require 'spec_helper'
 require 'digest/md5'
 
+module Aws
+  module SQS
+    class DummyClient
+      def send_message(params = {}, options = {})
+        nil
+      end
+      def receive_message(params = {})
+        []
+      end
+      def delete_message(params = {})
+        Seahorse::Client::Response.new
+      end
+      def delete_message_batch(params = {})
+        Aws::SQS::Types::DeleteMessageBatchResult.new
+      end
+    end
+
+    module Types
+      class DummyReceiveMessageResult
+        def initialize
+          @msg = []
+          @msg.push(DummyMessage.new)
+        end
+        def messages
+          @msg
+        end
+      end
+
+      class DummyMessage
+        attr_accessor :receipt_handle
+      end
+    end
+  end
+
+  module S3
+    class DummyClient
+      def put_object(params = {})
+
+      end
+      def get_object(params = {})
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)